cgroup2/manager_test.go

```diff
@@ -42,6 +42,7 @@ func TestEventChanCleanupOnCgroupRemoval(t *testing.T) {
 	proc := cmd.Process
 	if proc == nil {
 		t.Fatal("Process is nil")
+		return
```

Comment:
why?

Reply:
This was an attempt to resolve SA5011 from staticcheck linter. (below) I am thinking staticcheck isn't smart enough to know `t.Fatal` calls `runtime.exit` so we wouldn't really hit this issue.

```
cgroup2/manager_test.go:47:56: SA5011: possible nil pointer dereference (staticcheck)
        group := fmt.Sprintf("testing-watcher-%d.scope", proc.Pid)
                                                              ^
cgroup2/manager_test.go:43:5: SA5011(related information): this check suggests that the pointer can be nil (staticcheck)
        if proc == nil {
           ^
cgroup2/manager_test.go:48:39: SA5011: possible nil pointer dereference (staticcheck)
        c, err := NewSystemd("", group, proc.Pid, &Resources{})
```